contents/blog/posthog-vs-pendo.md

```diff
@@ -0,0 +1,646 @@
+---
+date: 2023-04-14
+title: PostHog vs Pendo
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-mixpanel/posthog-vs-mixpanel.jpeg
+featuredImageType: full
+author:
+    - joe-martin
+category: General
+tags:
+    - Comparisons
+    - Guides
+---
+
+In this article we’ll explore the crucial differences and similarities between Pendo (specifically, Pendo Engage) and PostHog. We’ll cover topics such as:
```

Comment:
Can add something similar to this at the start?

![Screenshot 2023-04-17 at 11 41 16](https://user-images.githubusercontent.com/92976667/232461726-959ee79f-6672-469a-8814-f1795cba1a67.png)

Reason being, Google loves to have an answer right at the top + it'll give us a featured snippet like this:

![Screenshot 2023-04-17 at 11 43 11](https://user-images.githubusercontent.com/92976667/232461923-22e380b9-537e-445c-85a4-cf9b806989d9.png)

Reply:
Done